hs_app_netCDF/models.py

```diff
@@ -290,57 +292,62 @@ def get_xml(self):
+            hsterms_ori_cov = etree.SubElement(container, '{%s}spatialReference' % self.NAMESPACES['hsterms'])
             hsterms_ori_cov_rdf_Description = etree.SubElement(hsterms_ori_cov, '{%s}Description' % self.NAMESPACES['rdf'])
             ori_cov_obj = self.ori_coverage.all().first()
 
+            # add extent info
             if ori_cov_obj.value:
-                cov_box = 'northlimit=%s; eastlimit=%s; southlimit=%s; westlimit=%s; units=%s' \
+                cov_box = 'northlimit=%s; eastlimit=%s; southlimit=%s; westlimit=%s; unit=%s' \
                         %(ori_cov_obj.value['northlimit'], ori_cov_obj.value['eastlimit'],
-                          ori_cov_obj.value['southlimit'], ori_cov_obj.value['westlimit'],ori_cov_obj.value['units'])
-                if 'projection' in ori_cov_obj.value:
-                    cov_box= cov_box + '; projection=%s' % ori_cov_obj.value['projection']
+                          ori_cov_obj.value['southlimit'], ori_cov_obj.value['westlimit'], ori_cov_obj.value['units'])
 
-                hsterms_ori_cov_box = etree.SubElement(hsterms_ori_cov_rdf_Description, '{%s}boundingBox' % self.NAMESPACES['hsterms'])
+                hsterms_ori_cov_box = etree.SubElement(hsterms_ori_cov_rdf_Description, '{%s}extent' % self.NAMESPACES['hsterms'])
                 hsterms_ori_cov_box.text = cov_box
 
-            # write projection string type and text info
+            # add crs info
+            if ori_cov_obj.value.get('projection'):
+                hsterms_ori_cov_projection_name = etree.SubElement(hsterms_ori_cov_rdf_Description,
+                                                                   '{%s}crsName' % self.NAMESPACES['hsterms'])
+                hsterms_ori_cov_projection_name.text = ori_cov_obj.value['projection']
+
             if ori_cov_obj.projection_string_text:
+                hsterms_ori_cov_projection_text = etree.SubElement(hsterms_ori_cov_rdf_Description, '{%s}crsRepresentationText' % self.NAMESPACES['hsterms'])
+                hsterms_ori_cov_projection_text.text = ori_cov_obj.projection_string_text
                 if ori_cov_obj.projection_string_type:
-                    hsterms_ori_cov_projection_type = etree.SubElement(hsterms_ori_cov_rdf_Description, '{%s}projectionStringType' % self.NAMESPACES['hsterms'])
+                    hsterms_ori_cov_projection_type = etree.SubElement(hsterms_ori_cov_rdf_Description, '{%s}crsRepresentationType' % self.NAMESPACES['hsterms'])
                     hsterms_ori_cov_projection_type.text = ori_cov_obj.projection_string_type
-                hsterms_ori_cov_projection_text = etree.SubElement(hsterms_ori_cov_rdf_Description, '{%s}projectionStringText' % self.NAMESPACES['hsterms'])
-                hsterms_ori_cov_projection_text.text = ori_cov_obj.projection_string_text
-
 
         return etree.tostring(RDF_ROOT, pretty_print=True)
 
+    def add_metadata_element_to_xml(self, root, md_element, md_fields):
```

Comment:
@gantian127 you don't have to add this function in your model since there is already such a function with the same name add_metadata_element_to_xml() available in hs_core/models.py that was added by Brian a while ago, so you can call that function directly in the same way as you are doing now without defining this function again in your specific models.py.